buildbot/buildbot/worker.py

```diff
@@ -375,10 +399,66 @@ def stage_hls(db, config):
             )
 
 
+def stage_afi(db, config):
+    """AFI stage: Create the AWS FPGA binary and AFI from the *.xclbin 
+    (Xilinx FPGA binary file)
+    """
+    with work(db, state.AFI_START, state.AFI, state.HLS_FINISH) as task:
+        # clean all the generated files from previous runs.
+        task.run(
+            ['sh', '-c', 'cd xclbin; \
+              rm -rf to_aws; rm -rf *afi_id.txt;\
+              rm -rf *.tar; rm -rf *afgi_id.txt'],
+            cwd=CODE_DIR
```

Comment:
Instead of using `cd xclbin ; ...` you can use `cwd=os.path.joing(CODE_DIR, 'xclbin')` here.